packages/webapp/src/webview/ui/components/App/App.scss

```diff
@@ -37,6 +37,7 @@ h1 {
     display: flex;
     width: 100%;
     flex-direction: column;
+    // overflow: auto;
```

Comment:
Is this a left over?

Reply:
yup, I'll remove it. Thanks for point it out 😊